contracts/payloads/IGP63/PayloadIGP63.sol

```diff
@@ -0,0 +1,397 @@
+    uint256 public constant PROPOSAL_ID = 62;
+
+    // State
+    uint256 public INST_ETH_VAULT_ID = 0;
+    uint256 public INST_ETH_DEX_ID = 0;
+    uint256 public ETH_USDC_DEX_ID = 0;
+    uint256 public ETH_USDC_VAULT_ID = 0;
+
+    function propose(string memory description) external {
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(this) == PROPOSER_AVO_MULTISIG_3 || 
+                address(this) == PROPOSER_AVO_MULTISIG_4,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Increase weETH-ETH_wstETH limits
+        action1();
+
+        // Action 2: Update fUSDC and fUSDT Rewards
+        action2();
+
+        // Action 3: Lend GHO to fGHO from Treasury
+        action3();  
+    }
+
+    function verifyProposal() external view {}
+
+
+    /**
+     * |
+     * |     Team Multisig Actions      |
+     * |__________________________________
+     */
+    function setState(
+        uint256 inst_eth_dex_id,
+        uint256 inst_eth_vault_id,
+        uint256 eth_usdc_dex_id,
+        uint256 eth_usdc_vault_id
+    ) external {
+        if (msg.sender != TEAM_MULTISIG) {
+            revert("not-team-multisig");
+        }
+
+        INST_ETH_DEX_ID = inst_eth_dex_id;
+        INST_ETH_VAULT_ID = inst_eth_vault_id;
+        ETH_USDC_DEX_ID = eth_usdc_dex_id;
+        ETH_USDC_VAULT_ID = eth_usdc_vault_id;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    /// @notice Action 1: Increase weETH-ETH_wstETH limits
+    function action1() internal {
+        address WEETH_ETH_WSTETH_DEX_ADDRESS = getDexAddress(9);
+        address WEETH_ETH_WSTETH_VAULT_ADDRESS = getVaultAddress(74);
+
+        { // Increase WEETH-ETH-WSTETH vault limit
+            IFluidAdminDex.UserSupplyConfig[] memory config_ = new IFluidDex.UserSupplyConfig[](1);
```

Comment:
on this, we actually need to increase the `updateMaxSupplyShares` on the Dex which are currently 4700, to 6000. 

We can also update the base withdrawal limit as you have here too, but I would only increase by the same ~27.6% from 2300 to 2935